### PR TITLE
Use [brackets] in non-literal glosses

### PR DESCRIPTION
**Pull request by lynn**
Also, move abbreviations to a new `gloss_abbreviation` field. 

Based on this discussion:

Hoemaı explains

### CODE REVIEW

File: tools/normalize.js [hunk 1]
```diff
@@ -65,6 +75,10 @@ d = d.map(obj => {
 
   obj_.gloss = obj_.gloss.replaceAll(' ', '.');
 
+  const remainingKeys = Object.keys(obj);
+  if (remainingKeys.length > 0) {
+    throw new Error(`word «${toaq}» has superfluous fields: ${JSON.stringify(remainingKeys)}`);
```
Comment (robintown) [inline tools/normalize.js:80]:
```suggestion
    throw new Error(`word «${toaq}» has superfluous fields: ${remainingKeys}`);
```

File: tools/normalize.js [hunk 2]
```diff
@@ -39,22 +39,32 @@ d = d.sort((a, b) => {
...
   ensureField(['gloss'], '');
   ensureField(['gloss_abbreviation']);
   ensureField(['short'], '');
   ensureField(['keywords'], []);
 
-  if([
-    'predicate', 'pronoun',    'object incorporating verb',
-    'name verb', 'word-quote', 'text-quote',
-  ].includes(obj.type)) {
+  if(verbyTypes.includes(type)) {
```
Comment (robintown) [inline tools/normalize.js:67]:
```suggestion
  if (verbyTypes.includes(type)) {
```

File: tools/normalize.js [hunk 3]
```diff
@@ -39,22 +39,32 @@ d = d.sort((a, b) => {
...
+];
+
 d = d.map(obj => {
   let obj_ = {};
-  let ensureField = (fields, orelse) => fields.forEach(field =>
-    obj_[field] = obj[field] || (typeof orelse === 'function' ? orelse(field) : orelse));
+  const type = obj.type;
+  const toaq = obj.toaq;
+  const ensureField = (fields, orelse) => fields.forEach(field => {
+    obj_[field] = obj[field] || (typeof orelse === 'function' ? orelse(field) : orelse)
```
Comment (robintown) [inline tools/normalize.js:56]:
```suggestion
    obj_[field] = obj[field] || (typeof orelse === 'function' ? orelse(field) : orelse);
```

**Review by robintown — APPROVED** [review]

**Review by uakci — APPROVED** [review]
